src/pages/Signup.jsx

```diff
@@ -29,28 +27,34 @@ import { PodRegistrationForm, ShowNewPod, initializePod, registerPod } from '@co
  * @returns {React.Component} - A react page
  */
 const Signup = () => {
-  const [oidcIssuer] = useState(ENV.VITE_SOLID_POD_SERVER || ENV.VITE_SOLID_IDENTITY_PROVIDER);
+  const [oidcIssuer] = useState(ENV.VITE_SOLID_IDENTITY_PROVIDER);
   const [searchParams] = useSearchParams();
   const caseManagerWebId = decodeURIComponent(searchParams.get('webId'));
   const [caseManagerName, setCaseManagerName] = useState();
-  const { session, login, podUrl } = useSession();
-  const { add: addContact, isSuccess } = useContactsList();
+  const [step, setStep] = useState('begin');
 
-  const registerAndLogin = async (email, password, confirmPassword) => {
-    await registerPod(
+  const registerAndInitialize = async (email, password, confirmPassword) => {
+    setStep('loading');
+    const registration = await registerPod(
       {
         email,
         password,
         confirmPassword
       },
       oidcIssuer
     );
-
-    await login({
-      oidcIssuer,
-      redirectUrl: window.location.href,
-      clientName: 'PASS Registration'
-    });
+    const caseManagerNames = caseManagerName?.split(' ') || [];
+    await initializePod(
+      registration.webId,
+      registration.podUrl,
+      {
+        caseManagerWebId,
+        caseManagerFirstName: caseManagerNames[0],
+        caseManagerLastName: caseManagerNames[caseManagerNames.length - 1]
+      },
+      registration.fetch
+    );
+    setStep('done');
   };
 
   const loadProfileInfo = async () => {
```

Comment:
Receiving this error when first clicking on Sign Up

<img width="217" alt="2024-02-25 (2)" src="https://github.com/codeforpdx/PASS/assets/83156697/87ebd8e4-08ac-444a-9fca-29c475075b3a">

Reply:
This should have only happened when clicking the signup button. Not when using the invite link.

This is now fixed.